dev/com.ibm.jbatch.container/resources/com/ibm/jbatch/container/internal/resources/JBatchMessages.nlsprops

```diff
@@ -290,4 +290,16 @@ info.batch.events.publish.topic.useraction=The batch JMS topic root can be modif
 
 cdi.ambiguous.artifact.names=CWWKY0041W: The batch container is running with a CDI feature active, but the CDI engine was not able to load the batch artifact as a CDI-managed bean and perform other CDI injections. This problem might be caused by a missing bean-defining annotation such as @Dependent, an unintended bean discovery mode, or an ambiguous 'ref' value.
 cdi.ambiguous.artifact.names.explanation=When the batch runtime loads artifacts by using the CDI engine, the batch properties (@BatchProperty) and batch contexts (JobContext and StepContext) are injected, and the full capabilities of CDI are made available. These capabilities include the injection of other managed beans according to qualifiers and all other features of the CDI programming model. When the batch runtime loads artifacts without using the CDI engine, it only injects the batch properties and contexts. Because it is not a CDI-managed bean in this path, no other injections or other features of the CDI programming model are available within that batch artifact. This message indicates that although the server is configured so that the CDI feature is active, the batch artifact cannot be loaded as a CDI-managed bean because the 'ref' attribute value that corresponds to this artifact reference in XML does not point to a single, unambiguous CDI-managed bean.
-cdi.ambiguous.artifact.names.useraction=Add the @Dependent annotation or another suitable bean-defining annotation to your batch artifact. Alternatively, you can set bean discovery mode to 'all' in the beans.xml file, although this solution might slow down your application. If all of the ambiguous artifacts are not needed, you can also remove the ambiguous 'ref' attributes from the application so that only a single, unambiguous reference remains.
\ No newline at end of file
+cdi.ambiguous.artifact.names.useraction=Add the @Dependent annotation or another suitable bean-defining annotation to your batch artifact. Alternatively, you can set bean discovery mode to 'all' in the beans.xml file, although this solution might slow down your application. If all of the ambiguous artifacts are not needed, you can also remove the ambiguous 'ref' attributes from the application so that only a single, unambiguous reference remains.
+
+#{0} job/partition
+#{1} The JSL
+display.resolved.jsl=CWWKY0042I:  The resolved JSL for this {0} is \n {1}.
+display.resolved.jsl.explanation=The JSL used for the job or partition is displayed after parameters and properties are resolved.  The job JSL will not resolve partition plan properties.  Look for this message in the individual partition logs for partition plan property resolution.
```

Comment:
Since the code shows us logging at the split-flow level, should this be:  "The JSL used for the job, flow, or partition..."  ? 

If translation has already gotten kicked off, though, it's fine.

Reply:
Thanks...I noticed that one myself and fixed it already..